pkg/terminal/terminal.go

```diff
@@ -268,12 +247,13 @@ func (t *Term) Run() (int, error) {
 }
 
 // Println prints a line to the terminal.
-func (t *Term) Println(prefix, str string) {
-	if !t.dumb {
-		terminalColorEscapeCode := fmt.Sprintf(terminalHighlightEscapeCode, t.conf.SourceListLineColor)
-		prefix = fmt.Sprintf("%s%s%s", terminalColorEscapeCode, prefix, terminalResetEscapeCode)
-	}
-	fmt.Fprintf(t.stdout, "%s%s\n", prefix, str)
+func (t *Term) Println(str string) {
+	fmt.Fprintf(t.stdout, "%s\n", str)
+}
+
+func (t *Term) PrintHighlightedCode(code string) error {
+	err := beni.Highlight(strings.NewReader(code), t.stdout, "Go", t.conf.SourceCodeStyle, "Terminal256")
```

Comment:
I don't think you can call this with a chunk of a file and expect it to work, the first line that we print could be inside a comment, for example.

Reply:
I tested it with a chunk starting with a commented line before. What could be the possible reason for failure?